scripts/opposed-wfrp4e.js

```diff
@@ -80,16 +81,17 @@ class OpposedWFRP
     {
       "flags.data.isOpposedTest": true
     });
-    this.createOpposedStartMessage(message.data.speaker);
+    this.createOpposedStartMessage(message.data.speaker, rollMode);
   }
 
   /**
    * Defender responds to an opposed test - evaluate result
-   * 
+   *
    * @param {Object} testResult Test result values
    * @param {Object} message message for update, actor, token, etc. for the defender
+   * @param {String} rollMode the type of roll mode used for the card
    */
-  static defenderClicked(testResult, message)
+  static defenderClicked(testResult, message, rollMode)
```

Comment:
rollMode doesn't appear to be used in this function

Reply:
yes, not use aymore.